frontend/src/container/LogsExplorerViews/index.tsx

```diff
@@ -665,6 +700,30 @@ function LogsExplorerViews({
 								</div>
 							</div>
 						)}
+						{(selectedPanelType === PANEL_TYPES.TIME_SERIES ||
+							selectedPanelType === PANEL_TYPES.TABLE) && (
+							<div className="query-stats">
+								<QueryStatus
+									loading={isLoading || isFetching}
+									error={isError}
+									success={isSuccess}
+								/>
+								{queryStats?.read_rows && (
+									<Typography.Text className="rows">
```

Comment:
Please make sure fallbacks are in place here. Bad data shouldn't break the UI

Reply:
if we do not receive any data we do not show anything in the UI, just the status of the request.